hugegraph-llm/src/hugegraph_llm/api/models/rag_requests.py

```diff
@@ -21,23 +21,29 @@
 
 
 class RAGRequest(BaseModel):
-    query: str
-    raw_llm: Optional[bool] = False
-    vector_only: Optional[bool] = False
-    graph_only: Optional[bool] = False
-    graph_vector: Optional[bool] = False
+    text: str = ""
+    raw_answer: bool = False
+    vector_only_answer: bool = False
+    graph_only_answer: bool = False
+    graph_vector_answer: bool = False
```

Comment:
why remove `Optional`?

Reply:
My reasons for removing optional:
The purpose of using optional is to make the value support None, but since I have assigned it by default, None will never happen, so the optional is effectively invalid, so it is removed

References:
1. Fastapi official documentation (https://fastapi.tiangolo.com/python-types/#using-union-or-optional)
2. To PyPI optional. The interpretation of the py (https://pypi.org/project/optional.py/)